crates/runtime/tests/acceleration/cron.rs

```diff
@@ -457,3 +436,121 @@ async fn test_cron_reload() -> Result<(), anyhow::Error> {
         })
         .await
 }
+
+const NAMES_TIMESTAMPED_CSV: &str = include_str!("data/names_timestamped.csv");
+
+#[tokio::test]
+#[allow(clippy::too_many_lines)]
+async fn test_append_cron_schedule() -> Result<(), anyhow::Error> {
+    let _tracing = init_tracing(Some("integration=debug,info"));
+
+    test_request_context()
+        .scope(async {
+            std::fs::write("./test_append_cron_schedule.csv", NAMES_TIMESTAMPED_CSV)
+                .expect("write file");
+
+            let mut dataset = get_dataset(
```

Comment:
If its easier you can use the `test_framework::app_utils::load_app_from_spicepod_str` method to define the spicepod using normal YAML, see https://github.com/spiceai/spiceai/blob/trunk/crates/runtime/tests/iceberg/connector/audit_log.rs#L36 for an example.